### PR TITLE
Add Q&A (Installation) discussion form

### PR DESCRIPTION
**Pull request by MohitGupta121**
## Explanation

Add Q&A (Installation) discussion form

- This PR does the following: This is so we get all the information required to solve installation problems.

## Essential Checklist

- [ ] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## Proof that changes are correct

Screenshot 2023-02-18 at 12 18 16 PM

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Review by MohitGupta121 — COMMENTED** [review]
As @gp201 suggest me in last welfare meeting to do something similar for oppia-android.
@seanlip @gp201 @BenHenning PTAL, I make form similar as per oppia-web.

File: .github/DISCUSSION_TEMPLATE/q-a-installation.yml
Comment (BenHenning) [inline .github/DISCUSSION_TEMPLATE/q-a-installation.yml]: This is potentially ambiguous since we cross-compile an Android app. This should probably ask about what OS their _development environment_ is using.

File: .github/DISCUSSION_TEMPLATE/q-a-installation.yml
Comment (BenHenning) [inline .github/DISCUSSION_TEMPLATE/q-a-installation.yml:17]: Free disk space would also be useful since builds require a lot of on-disk storage.

**Review by BenHenning — COMMENTED** [review]
Thanks @MohitGupta121! This is interesting. Left a few comments after taking a first pass, PTAL.

File: .github/DISCUSSION_TEMPLATE/q-a-installation.yml
Comment (MohitGupta121) [reply .github/DISCUSSION_TEMPLATE/q-a-installation.yml:17]: Need to add new field of "Free Disk Space on-disk" ?

File: .github/DISCUSSION_TEMPLATE/q-a-installation.yml
Comment (MohitGupta121) [reply .github/DISCUSSION_TEMPLATE/q-a-installation.yml]: Done.

File: .github/DISCUSSION_TEMPLATE/q-a-installation.yml
Comment (MohitGupta121) [reply .github/DISCUSSION_TEMPLATE/q-a-installation.yml:17]: Done.

**Comment by seanlip** [comment]
@MohitGupta121 I don't think you need a review from me on this. Please check with @gp201 on any dev-workflow-specific issues you run into, but otherwise, once @BenHenning approves this I think you can merge it.

Thanks!

**Comment by MohitGupta121** [comment]
@seanlip okay thanks, Waiting for @BenHenning and @gp201 review.

**Review by gp201 — APPROVED** [review]
LGTM

**Review by BenHenning — APPROVED** [review]
Thanks @MohitGupta121. I think this looks really good! Excited to see how it works in practice.

**Comment by BenHenning** [comment]
Though it looks like you have a CI failure @MohitGupta121. Please fix & assign back.

**Comment by MohitGupta121** [comment]
> Though it looks like you have a CI failure

Yes @BenHenning I see the checks are failing saying due to CODEOWNER files. So how I can passed that test? 
Need to change anything in CODEOWNER file?

I think we need to need CODEOWNER for this file mentioned in the CI Check [Failure](https://github.com/oppia/oppia-android/actions/runs/4225693812/jobs/7338291938#step:4:16) 

What do you suggest Ben is that correct, who is the CODEOWNER?

**Comment by MohitGupta121** [comment]
@BenHenning PTAL, all checks are passed now.

**Review by BenHenning — APPROVED** [review]
Thanks @MohitGupta121. This LGTM!

**Pull request merged**
